wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/core-customizer-css.core-4.9.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => '53951abcc030599f547d879bd9566da0');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).